test/unit/mdc-drawer/mdc-drawer.test.js

```diff
@@ -27,14 +27,20 @@ import domEvents from 'dom-events';
 import td from 'testdouble';
 
 import {MDCDrawer} from '../../../packages/mdc-drawer';
-import {strings} from '../../../packages/mdc-drawer/constants';
+import * as util from '../../../packages/mdc-drawer/util';
+import {strings, cssClasses} from '../../../packages/mdc-drawer/constants';
 import {MDCListFoundation} from '../../../packages/mdc-list';
 import MDCDismissibleDrawerFoundation from '../../../packages/mdc-drawer/dismissible/foundation';
 
-function getFixture() {
+function getFixture(variantClass) {
+  let scrimEl;
```

Comment:
Should this be initialized to `''`? Will it produce `'undefined'` below for dismissible? (Though I suppose it won't affect anything)